### PR TITLE
Refactor example scripts and config.

### PR DESCRIPTION
**Pull request by heathkit**
Protractor is now only run using either 'npm run e2e' or 'npm run e2e-debug'

The script for the debug example echos what it's doing before running.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "test": "ng test",
     "lint": "ng lint",
     "e2e": "npm run webdriver-update && protractor",
```
Comment (cnishina) [inline package.json]: remove npm run webdriver-update before running protractor

File: scripts/update-goldens.sh [hunk 2]
```diff
@@ -1,10 +1,8 @@
 #!/usr/bin/env bash
```
Comment (cnishina) [inline scripts/update-goldens.sh]: move scripts from scripts/ to root directory

File: examples/1-screenshots.sh [hunk 3]
```diff
@@ -1,10 +1,8 @@
 #!/usr/bin/env bash
```
Comment (cnishina) [inline examples/1-screenshots.sh]: move scripts from examples/ to root directory

File: package.json [hunk 4]
```diff
@@ -9,7 +9,8 @@
...
-    "webdriver-update": "webdriver-manager update --versions.chrome=2.36 --gecko false"
+    "e2e-debug": "npm run webdriver-update && node --inspect $(npm bin)/protractor protractor-debug.conf.js",
```
Comment (cnishina) [inline package.json]: remove npm run webdriver-update from e2e-debug

File: package.json [hunk 5]
```diff
@@ -9,7 +9,8 @@
...
+    "webdriver-update": "webdriver-manager update --gecko false"
```
Comment (cnishina) [inline package.json]: do we know if a new version of chromedriver works? versions.chrome 2.36 worked but 2.37 did not.

**Review by cnishina — CHANGES_REQUESTED** [review]

File: scripts/update-goldens.sh (hunk 2 above)
Comment (heathkit) [reply scripts/update-goldens.sh]: done

File: package.json (hunk 5 above)
Comment (heathkit) [reply package.json]: How did it fail before? It seems to be working for me.

File: package.json (hunk 4 above)
Comment (heathkit) [reply package.json]: done

File: examples/1-screenshots.sh (hunk 3 above)
Comment (heathkit) [reply examples/1-screenshots.sh]: done

File: package.json (hunk 1 above)
Comment (heathkit) [reply package.json]: done

**Comment by heathkit** [comment]
Whoops, accidentally had this rebased onto some style changes I pushed....

**Pull request closed without merging**